conslogging/conslogging.go

```diff
@@ -91,35 +93,37 @@ func New(w io.Writer, mu *sync.Mutex, colorMode ColorMode, prefixPadding int, lo
 		mu = &sync.Mutex{}
 	}
 	return ConsoleLogger{
-		consoleErrW:    w,
-		errW:           w,
-		colorMode:      colorMode,
-		saltColors:     make(map[string]*color.Color),
-		nextColorIndex: new(int),
-		prefixPadding:  prefixPadding,
-		mu:             mu,
-		logLevel:       logLevel,
+		consoleErrW:     w,
+		errW:            w,
+		colorMode:       colorMode,
+		saltColors:      make(map[string]*color.Color),
+		nextColorIndex:  new(int),
+		prefixPadding:   prefixPadding,
+		mu:              mu,
+		logLevel:        logLevel,
+		isGitHubActions: os.Getenv("GITHUB_ACTIONS") == "true",
```

Comment:
Wondering if we should pull this detection in the main package, as a flag that the user could override.

Reply:
I think that would be nice -- maybe a user wants to see the output occur when they are running it locally?

Reply:
Added as `--github-actions`. Not sure if there's a better name.

Reply:
maybe `--github-actions-annotations` or simply `--github-annotations`? I think it gives a better hint that this is specifically for producing annotations.